make_profiler/report/script.js

```diff
@@ -0,0 +1,148 @@
+                if (txtValue.toUpperCase().indexOf(filter) > -1) {
+                    tr[i].style.display = "";
+                } else {
+                    tr[i].style.display = "none";
+                }
+            }
+        }
+    }
+}
+
+//refresh page every 5 minutes
+setTimeout(function () {
+    window.location.reload(1);
+}, 1000 * 5 * 60);
+
+getStatus("report.json").then((resp) => {
+    let recs = resp.status;
+    let pipeline = resp.pipeline;
+    var pipelineTable =
+        `
+                    <div id="statusReport">
+                    <img class="img" src='report/Kontur_logo_main.png' />
+                        <div class="vertical-center">
+                            <p>PIPELINE DASHBOARD</p>
+                        </div>
+                    </div>
+                    <h2>Pipeline Status</h2>
+                    <table id="pipelineTable">
+                        <tr>
+                            <td><b>Present Status</b></td>
+                            <td align="center"><b>` + pipeline.presentStatus + `</b></td>
+                        </tr>
+                        <tr>
+                            <td>Targets total</td>
+                            <td align="center">` + pipeline.nEvents + `</td>
+                        </tr>
+                        <tr>
+                            <td>Targets in progress</td>
+                            <td align="center">` + pipeline.nProgress + `</td>
+                        </tr>
+                        <tr>
+                            <td>Targets frozen</td>
+                            <td align="center">0</td>
+                        </tr>
+                        <tr>
+                            <td>Targets failed</td>
+                            <td align="center">` + pipeline.nFail + `</td>
+                        </tr>
+                        <tr>
+                            <td>Oldest completed target</td>
+                            <td align="center">` + formatDate(pipeline.oldestCompleteTime) + `</td>
+                        </tr>
+                    </table>
+                    <h2>Target status</h2><a id="statusChart" target="_blank" href="make.svg">Status Chart</a></br>
+                    <input type="text" id="myInput" onkeyup="filterTarget()" placeholder="Search for target name.." title="Type in a name" />`
+
+
+    let statusTable = `
+                    <table id="statusTable" class="sortable">
+                        <tr class="header">
+                                    <th>Target name</th>
+                                    <th>Last completed date</th>
+                                    <th>Status</th>
+                                    <th>Status date</th>
+                                    <th>Duration</th>
+                                    <th>Log</th>
+                        </tr>`
+
+    for (var i = 0; i < recs.length; i++) {
+        statusTable += "<tr class=" + recs[i].eventType + ">";
```

Comment:
it's also much easier to read 
statusTable += `<tr class="${recs[i].eventType}">`;